app_dart/lib/src/request_handlers/reset_prod_task.dart

```diff
@@ -66,6 +72,22 @@ class ResetProdTask extends ApiRequestHandler<Body> {
           .map((LuciBuilder builder) => builder.name)
           .single;
     }
+    final RepositorySlug slug = RepositorySlug(owner, repo);
+    final Iterable<Build> currentBuilds = await luciBuildService.getProdBuilds(slug, commit.sha, builder, repo);
+    final List<Status> noReschedule = <Status>[Status.started, Status.scheduled, Status.success];
+    final Build build = currentBuilds.firstWhere(
+      (Build element) {
+        log.info('Found build status: ${element.status} inNoReschedule ${noReschedule.contains(element.status)}');
+        return noReschedule.contains(element.status);
+      },
+      orElse: () => null,
+    );
+    log.info(
+        'Owner: $owner, Repo: $repo, Builder: $builder, CommitSha: ${commit.sha}, CurrentBuild: $build, CurrentBuild: $currentBuilds');
```

Comment:
Nit: Build: $build, CurrentBuilds: $currentBuilds

Reply:
Updated and removed currentBuilds as it is just a list and its elements are being logged above.